dev-python/qdarkstyle/qdarkstyle-2.7.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+inherit distutils-r1
+
+MY_PN=QDarkStyleSheet
+
+DESCRIPTION="A dark style sheet for QtWidgets application"
+HOMEPAGE="https://github.com/ColinDuquesnoy/QDarkStyleSheet"
+SRC_URI="https://github.com/ColinDuquesnoy/${MY_PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+IUSE="doc develop examples test"
+
+RDEPEND=">=dev-python/helpdev-0.6.2[${PYTHON_USEDEP}]
+	develop? ( dev-python/qtsass[${PYTHON_USEDEP}]
+			dev-python/watchdog[${PYTHON_USEDEP}] )
+	doc? ( dev-python/sphinx[${PYTHON_USEDEP}]
+			dev-python/m2r[${PYTHON_USEDEP}]
+			dev-python/sphinx_rtd_theme[${PYTHON_USEDEP}] )
+	examples? ( >=dev-python/QtPy-1.7[${PYTHON_USEDEP}] )"
+
+DEPEND="test? ( dev-python/QtPy[${PYTHON_USEDEP}] )"
+
+distutils_enable_tests pytest
+
+S=${WORKDIR}/${MY_PN}-${PV}
```

Comment:
`S="..."`, the WORKDIR fails here without quotation marks if someone has a space in their path.